cirq/ops/moment.py

```diff
@@ -486,6 +486,9 @@ def _commutes_(
 
         return True
 
+    def _decompose_(self) -> 'cirq.OP_TREE':
+        return self._operations
+
```

Comment:
`cirq.decompose(moment)` already has a defined behavior - it decomposes each operation in the moment in sequence and returns the result as a list.
https://github.com/quantumlib/Cirq/blob/e3c673fa103db7c46673d4cd66558405c8b1218e/cirq/protocols/decompose_protocol.py#L239-L241
Defining this here is a breaking change.

Reply:
Oh wait, it turns out this wasn't even necessary. The first time I tried it I'd forgotten to add the `decompose` line to the simulator itself. Pushed new code without it.